### PR TITLE
Add review videos for AVL trees, code examples

### PR DESCRIPTION
**Pull request by msambol**
- Add review playlist for AVL trees
- Add code examples for review videos
- Update channel URL to the "friendly" one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1199,8 +1199,9 @@ Graphs can be used to represent many problems in computer science, so this secti
 
 - [ ] Series of 2-3 minutes short subject videos (23 videos)
     - [Videos](https://www.youtube.com/watch?v=r4r1DZcx1cM&list=PLmVb1OknmNJuC5POdcDv5oCS7_OUkDgpj&index=22)
-- [ ] Series of 2-5 minutes short subject videos - Michael Sambol (43 videos):
-    - [Videos](https://www.youtube.com/channel/UCzDJwLWoYCUQowF_nG3m5OQ)
+- [ ] Series of 2-5 minutes short subject videos - Michael Sambol (46 videos):
+    - [Videos](https://www.youtube.com/c/MichaelSambol)
```
Comment (msambol) [inline README.md:1203]: Just updating the video URL to the "friendly" one.

**Comment by jwasham** [comment]
It's a lot of stuff here. Let me know once you're ready to review.

**Comment by msambol** [comment]
> It's a lot of stuff here. Let me know once you're ready to review.

It's ready whenever you have time, John. It's the same three changes across every file. 

1. add AVL tree review playlist
2. update video count
3. use friendly channel url

**Comment by msambol** [comment]
Sorry for all the pushes 😅

**Comment by msambol** [comment]
@jwasham Gentle poke on this... anything you want me to address?

**Comment by jwasham** [comment]
Just gotta get around to it. Soon!

**Comment by jwasham** [comment]
The code examples link has links that point to videos and other resources, but I don't see code examples.

**Comment by msambol** [comment]
> The code examples link has links that point to videos and other resources, but I don't see code examples.

The code is in the folders and matches up to each of the videos. I can make that more clear somehow?

**Comment by jwasham** [comment]
I see. I was looking at the links only. Maybe a "(code)" link next to each? It's your repo. Just wanna make sure it will be helpful for visitors.

**Comment by msambol** [comment]
@jwasham Take a look now? Great feedback, thank you—I think it's easier to follow.

**Comment by jwasham** [comment]
Wow nice!

Thanks for this. Merging...

**Pull request merged**